modules/installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -19,6 +19,48 @@ As a developer, you can deploy a {product} instance on {ocp-short} by using the
 . From the *Developer* perspective on the {ocp-short} web console, click *+Add*.
 . From the *Developer Catalog* panel, click *Operator Backed*.
 . In the *Filter by keyword* box, enter _{product-short}_ and click the *{product}* card.
+. Create a config map named `app-config-rhdh` containing the {product-short} configuration using the following template:
```

Comment:
For consistency, consider that these steps should be exactly the same as the steps found in `modules/configuring/proc-provisioning-your-custom-configuration.adoc`.